mcp-server/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "open-digger-mcp-server",
+  "version": "1.0.0",
+  "description": "MCP server for OpenDigger data with comprehensive analysis tools, trend analysis, repository comparison capabilities and more.",
+  "license": "Apache-2.0",
+  "author": "Frank Zhao",
+  "contributors": [
+    {
+      "name": "Pratyay Banerjee"
+    }
+  ],
+  "homepage": "https://open-digger.cn/",
+  "bugs": "https://github.com/X-lab2017/open-digger-mcp-server/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/X-lab2017/open-digger-mcp-server.git"
+  },
+  "type": "module",
+  "bin": {
+    "open-digger-mcp-server": "dist/index.js"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsc && chmod +x dist/index.js",
+    "dev": "npm run build && node dist/index.js",
+    "start": "node dist/index.js",
+    "watch": "tsc --watch",
+    "clean": "rm -rf dist",
+    "prepare": "npm run build",
+    "sse:test": "SSE_PORT=3001 npm start",
+    "debug": "NODE_OPTIONS='--inspect' npm start"
```

Comment:
`npm start NODE_OPTIONS='--inspect'`

Reply:
You're right. The solution is the same as [above](https://github.com/X-lab2017/open-digger-mcp-server/pull/1#discussion_r2384203004).

Reply:
The command here needs to be replaced with `"NODE_OPTIONS='--inspect' npm start"`.

Reply:
@birdflyi Fixed via [`2997217`](https://github.com/X-lab2017/open-digger-mcp-server/pull/1/commits/2997217065d2e6bd9d366a7b3dc40e047ced0abc) 🚀